src/map/clif.c

```diff
@@ -19076,6 +19100,147 @@ const char *clif_get_bl_name(const struct block_list *bl)
 	return name;
 }
 
+/**
+ * Clan System: Sends the basic clan informations to client.
+ * 098a <length>.W <clan id>.L <clan name>.24B <clan master>.24B <clan map>.16B <alliance count>.B
+ *      <antagonist count>.B { <alliance>.24B } * alliance count { <antagonist>.24B } * antagonist count (ZC_CLANINFO)
+ */
+void clif_clan_basicinfo(struct map_session_data *sd)
+{
+#if PACKETVER >= 20120716
+	int fd, offset, length, i;
+	struct clan *c, *ally, *antagonist;
+	char mapname[MAP_NAME_LENGTH_EXT];
+
+	nullpo_retv(sd);
+	nullpo_retv(c = sd->clan);
+
+	length = 8 + 2 * NAME_LENGTH + MAP_NAME_LENGTH_EXT + 2;
+	fd = sd->fd;
+
+	WFIFOHEAD(fd, length);
+	memset(WFIFOP(fd, 0), 0, length);
+	WFIFOW(fd, 0) = 0x98a;
```

Comment:
This should be serialized through a (packed) structure like our other map packets?

Reply:
Is it better? If so, I can change it.